suite-common/wallet-core/src/device/deviceActions.ts

```diff
@@ -29,6 +29,16 @@ const deviceDisconnect = createAction(DEVICE.DISCONNECT, (payload: TrezorDevice)
     payload,
 }));
 
+// this action is not used but is required because of the typings
```

Comment:
This is awkward... how should it be resolved, @peter-sanderson? I see you removed `DeviceEvent` [here](https://github.com/trezor/trezor-suite/commit/c02412bccf80da7c827f624b7a7c85cdedf278c5#diff-2e9d057f0bfe2cc92fe50d4ce28838622d9e79fcca010ab8847a0fa288da13fd)

Reply:
mapping is done only for 2 actions however the rest of DEVICE_EVENTs are dispatched directly [from here](https://github.com/trezor/trezor-suite/blob/develop/suite-common/connect-init/src/connectInitThunks.ts#L43) not from the deviceActions, yet its required to be defined in deviceActions to satisfy the types.

this problem applies to any other DEVICE_EVENTs  action.type like DEVICE.DISCONNECT or DEVICE.PASSPHRASE or DEVICE.PIN

Reply:
what is the benefit of doing things like this? 
what i see is 1000% more lines of code, more complexity and added another todo's which will be forgotten until next the occurrence

Reply:
The idea is to be explicit and to unify syntax across the codebase. However, the current implementation is not finished and having to create actions that are not dispatched is indeed counter-intuitive. @peter-sanderson will try to finish the work he started in his [branch](https://github.com/trezor/trezor-suite/commit/29aaaf42f0ae2b3b706f35309dfad9a664b4aba1) - should he not succeed, I'll broaden the type again to remove the counter-intuitive behaviour.

Reply:
so in the future you want to duplicate all the definitions of the events emitted from TrezorConnect? 
like for example [40+ ui-events](https://github.com/trezor/trezor-suite/blob/develop/packages/connect/src/events/ui-request.ts)?

this is how undestand this comment:

```
TrezorConnectEvents // Todo: This should not be here, actions shall be defined independently from Connect Events (and they shall be mapped onto them)
```

Reply:
I agree this is too many events to map.